internal/scripts/prune-preview-deploys.ts

```diff
@@ -85,56 +80,47 @@ async function listPreviewWorkerDeployments() {
 	)
 }
 
-async function deleteQueue(id: string) {
-	await cloudflareApi(`/queues/${id}`, { method: 'DELETE' })
+async function deleteQueue(queueName: string) {
+	nicelog('Deleting queue:', queueName)
+	await exec('npx', ['wrangler', 'queues', 'delete', queueName], {
+		env: { CI: '1' },
+	})
 }
 
-async function deleteQueueConsumer({ id, consumerId }: { id: string; consumerId: string }) {
-	await cloudflareApi(`/queues/${id}/consumers/${consumerId}`, { method: 'DELETE' })
+async function deleteQueueConsumer(queueName: string, scriptName: string) {
+	nicelog('Deleting queue consumer:', scriptName, 'from queue:', queueName)
+	await exec('npx', ['wrangler', 'queues', 'consumer', 'worker', 'remove', queueName, scriptName], {
+		env: { CI: '1' },
+	})
 }
 
-async function deletePreviewWorker(id: string) {
-	const endpoint = `/workers/scripts/${id}`
-	nicelog(
-		'DELETE',
-		`https://api.cloudflare.com/client/v4/accounts/${env.CLOUDFLARE_ACCOUNT_ID}${endpoint}`
-	)
-	const res = await cloudflareApi(endpoint, { method: 'DELETE' })
-
-	if (!res.ok) {
-		throw new Error('Failed to delete worker ' + JSON.stringify(await res.json()))
-	}
+async function deletePreviewWorker(workerName: string) {
+	nicelog('Deleting worker:', workerName)
+	await exec('npx', ['wrangler', 'delete', '--name', workerName], {
+		env: { CI: '1' },
+	})
 }
 
 async function deletePreviewWorkerDeployment(id: string) {
 	// We want to delete the queue consumer and the queue only once. We'll do it just before we delete the worker
 	if (id.match(CLOUDFLARE_SYNC_WORKER_REGEX)) {
 		const prNumber = Number(id.match(CLOUDFLARE_WORKER_REGEX)?.[1])
 		const queueName = `tldraw-multiplayer-queue-pr-${prNumber}`
-		const queueInfo = queuesMap.get(queueName)
-		if (queueInfo) {
-			const { id, consumerId } = queueInfo
-			if (consumerId) {
-				await deleteQueueConsumer({ id, consumerId })
-			}
-			await deleteQueue(id)
-		}
-	}
-	await deletePreviewWorker(id)
-}
-
-const queuesMap = new Map<string, { id: string; consumerId: string | undefined }>()
 
-async function getQueues() {
```

Comment:
Deleting via the api required consumer id, where as wrangler allows using the worker id, so this is no longer needed.